speedy-recovery-client/src/components/secured/patientinformation/Practitioner/PractitionerPatientInfo.js

```diff
@@ -0,0 +1,213 @@
+import "./PractitionerInfo.css";
+import PractitionerConditionPane from "./PractitionerConditionPane";
+import PractitionerCarePlanPane from "./PractitionerCarePlanPane";
+import PractitionerMedicationDispensePane from "./PractitionerMedicationDispensePane";
+import PractitionerBasicPane from "./PractitionerBasicPane";
+import PractitionerFamilyHistory from "./PractitionerFamilyHistory";
+import PractitionerGoal from "./PractitionerGoal";
+import {
+  fhirMapCarePlan,
+  fhirMapCondition,
+  fhirMapMedicationDispense,
+  fhirMapFamilyResource,
+  fhirMapGoal
+} from "../../../../service/FhirDataMappingService";
+
+class PractitionerPatientInfo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      patient: {},
+      conditions: [],
+      medicationDispenses: [],
+      carePlans: [],
+      histories: [],
+      goal: []
+    };
+  }
+
+  render() {
+    const panes = [
+      {
+        menuItem: (
+          <Menu.Item key={"basic"}>
+            <Icon fitted name="id card outline" color="blue" size="large" />
+            Basic
+          </Menu.Item>
+        ),
+
+        render: () => (
+          <PractitionerBasicPane user={this.props.location.state.patient} />
+        )
+      },
+      {
+        menuItem: (
+          <Menu.Item key={"medication"}>
+            <Icon fitted name="pills" color="purple" size="large" />
+            Dispensed Medication
+          </Menu.Item>
+        ),
+        render: () => (
+          <PractitionerMedicationDispensePane
+            medicationDispenses={this.state.medicationDispenses}
+          />
+        )
+      },
+      {
+        menuItem: (
+          <Menu.Item key={"condition"}>
+            <Icon fitted name="heartbeat" color="red" size="large" />
+            Condition
+          </Menu.Item>
+        ),
+        render: () => (
+          <PractitionerConditionPane conditions={this.state.conditions} />
+        )
+      },
+      {
+        menuItem: (
+          <Menu.Item key={"carePlan"}>
+            <Icon fitted name="unordered list" color="orange" size="large" />
+            Care Plan
+          </Menu.Item>
+        ),
+        render: () => <PractitionerCarePlanPane carePlans={this.state.carePlans} />
+      },
+
+      {
+        menuItem: (
+          <Menu.Item key={"histories"}>
+            <Icon fitted name="book" color="green" size="large" />
+            Family History
+          </Menu.Item>
+        ),
+        render: () => (
+          <PractitionerFamilyHistory histories={this.state.histories} />
+        )
+      },
+
+      {
+        menuItem: (
+          <Menu.Item key={"goal"}>
+            <Icon fitted name="trophy" color="yellow" size="large" />
+            Goal
+          </Menu.Item>
+        ),
+
+        render: () => <PractitionerGoal goal={this.state.goal} />
+      }
+    ];
+
+    return (
+      <div>
+        <h1>Patient Information For Practitioner View</h1>
```

Comment:
you can remove this line and just say "information for {patientName}". You can also remove the next line then